frontend/public/locales/en/translation.json

```diff
@@ -2115,7 +2116,8 @@
     "submariner.install.form.gateways.labelHelp": "The number of worker nodes  that will be used to deploy the Submariner gateway component on the managed cluster. If the value is greater than 1, the Submariner gateway HA will be enabled automatically. (default 1)",
```

Comment:
Change to:

The number of worker nodes used to deploy the Submariner gateway component on the managed cluster. If the value is greater than 1, the Submariner gateway HA is enabled automatically (default 1).

Also, what is HA?

Reply:
Done, HA is high availablity